### PR TITLE
Support for target groups (fix for #308)

### PR DESCRIPTION
**Pull request by msiuts**
This adds support for targetGroups on AutoScalingGroups which is a fix for issue #308

### CODE REVIEW

**Comment by lanej** [comment]
@msiuts could you add that additional entry to [the format tests](https://github.com/fog/fog-aws/blob/09a405cc1474636f436406fc31c7b8baebaead8f/tests/requests/auto_scaling/helper.rb#L82) please?

File: lib/fog/aws/parsers/auto_scaling/describe_auto_scaling_groups.rb [hunk 1]
```diff
@@ -21,7 +21,8 @@ def reset_auto_scaling_group
               'LoadBalancerNames' => [],
               'SuspendedProcesses' => [],
               'Tags' => [],
-              'TerminationPolicies' => []
+              'TerminationPolicies' => [],
+              'TargetGroupARNs' => []
```
Comment (lanej) [inline lib/fog/aws/parsers/auto_scaling/describe_auto_scaling_groups.rb]: http://docs.aws.amazon.com/AutoScaling/latest/APIReference/API_AutoScalingGroup.html

**Comment by msiuts** [comment]
@lanej is this now fine with you?

**Comment by lanej** [comment]
thanks @msiuts

**Comment by geemus** [comment]
Thanks!

**Pull request merged**